sdk/identity/perf-tests/identity/test/DeviceCodeCredential/persistence.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { PerfStressTest } from "@azure/test-utils-perfstress";
+import { DeviceCodeCredential } from "@azure/identity";
+
+const scope = `https://servicebus.azure.net/.default`;
+
+export abstract class DeviceCodeCredentialTest extends PerfStressTest {
+  options = {};
+  credential: DeviceCodeCredential;
+
+  constructor() {
+    super();
+    this.credential = new DeviceCodeCredential({
```

Comment:
replace this with Client Secret Credential so that it is easy to automate.

Reply:
Can do! Anything else?

Reply:
nope, all fine otherwise, as long as perf test compiles and runs.

Reply:
@g2vinay the test does compile and run, but we can't make this run on CI because we currently can't have `@azure/msal-node-extensions@1.0.0-alpha.6` as a dependency, at the moment 😵

Reply:
Now I'm using ClientSecretCredential :)